### PR TITLE
Add working directory related information to debugger guide

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will add working directory related information to debugger guide.

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/29480

### CODE REVIEW

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ The VS Code Ballerina extension gives you the same debugging experience as the c
 
 Then, you can start a program, test, or remote debug session as shown below.
 
+>**Info:** The working directory will always be the package root if the user starts the debug session through VS Code. Users can use remote debugging for alternative working directories.
```
Comment (NipunaRanasinghe) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]:
```suggestion
>**Info:** For the debug sessions which get launched from the VS Code, the working directory will be the Ballerina package root. Users can use remote debugging for alternative working directories.
```

Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]: @praneesha Appreciate your input on this

Comment (praneesha) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]:
```suggestion
>**Info:** If you launch the debug session through VS Code, the working directory will always be the package root. However, you can use remote debugging for alternative working directories.
```

Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]: >

Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]: @praneesha How about `>**Info:** If you launch the debug session through VS Code, the working directory will be the Ballerina package root. However, you can use remote debugging for alternative working directories.`

Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]: Replied below.

Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:65]: +1

Comment (praveennadarajah) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md]: Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/2151/commits/9638a4849dbb2b96e940cfa945c9e12d58c00b23

**Review by praneesha — APPROVED** [review]

**Pull request merged**